exalearn-ui/src/app/header/header.component.less

```diff
@@ -0,0 +1,48 @@
+.header {
+  position: fixed;
+  width: 100vw;
+  background-color: #e9e9e9;
```

Comment:
Please, change .less to .scss and refactor styles in according to this format.
Note: After @GeranSkot will merge his PR, all colors we have to replace by constants from styles/colors.scss